### DISCUSSION TITLE
Veracidade dos dados do CNPJ

### DISCUSSION THREAD

**Discussion by dariokrugerjunior** [BrasilAPI/BrasilAPI#428] [Q&A] [answered]
Queria saber qual é a veracidade dos dados da requisição pelo cnpj ? se ele é atualizado em algum período ?

**Comment by RenatoOliveiraS** [comment]
Tambem gostaria de saber! Os arquivos disponibilizados pela receita agora são mensais.

**Comment by murilohns** [comment]
nós pegamos os dados [dessa api](https://docs.minhareceita.org/)

**Reply by RodriAndreotti** [reply]
Então, eu já ajudei o @cuducos A baixar esses arquivos uma vez, ele baixa todo santo mês para atualizar a base.

**Comment by cuducos** [comment, marked as answer]
> Queria saber qual é a veracidade dos dados da requisição pelo cnpj ?

https://docs.minhareceita.org/sobre-os-dados/

> se ele é atualizado em algum período ?

Atualizo sempre que a Receita Federal solta dados novos. Você pode acompanhar no _enpoint_ específico para isso: https://docs.minhareceita.org/como-usar/#endpoints-auxiliares

**Comment by vito-ysl** [comment]
Os dados obtidos por APIs de CNPJ (como a da Receita Federal ou de projetos abertos) são geralmente baseados nos dados públicos da Receita Federal.
